doc/v2.1-joss.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'khmer release v2.1: software for biological sequence analysis'
+tags:
+    - bioinformatics
+    - sequence analysis
+    - Bloom filter
+    - Count-Min sketch
+    - de Bruijn graph
+    - assembly
+    - graph traversal
+authors:
+    - name: Daniel Standage
+      orcid: 0000-0003-0342-8531
+      affiliation: 1
+    - name: Ali Aliyari
+      orcid:
+      affiliation: 2
+    - name: Lisa J. Cohen
+      orcid:
+      affiliation:
+          - 1
+          - 3
+    - name: Michael R. Crusoe
+      orcid: 0000-0002-2961-9670
+      affiliation: 4
+    - name: Tim Head
+      orcid: 0000-0003-0931-3698
+      affiliation: 5
+    - name: Luiz Irber
+      orcid:
+      affiliation:
+          - 1
+          - 6
+    - name: Shannon Joslin
+      orcid:
+      affiliation: 2
+    - name: Nicole Kingsley
+      orcid:
+      affiliation: 2
+    - name: Kevin D. Murray
+      orcid: 0000-0002-2466-1917
+      affiliation: 7
+    - name: Camille Scott
+      orcid:
+      affiliation:
+          - 1
+          - 6
+    - name: Sascha Steinbiss
+      orcid:
+      affiliation: 8
```

Comment:
This is no longer my current affiliation; I left the Institute in October 2016 to continue my work in a field unrelated to bioinformatics. In this light it would probably be better to include the Debian Project as my affiliation.